libmount/src/mountP.h

```diff
@@ -93,7 +94,25 @@ struct libmnt_test {
 extern int mnt_run_test(struct libmnt_test *tests, int argc, char *argv[]);
 #endif
 
+#ifdef UL_HAVE_MOUNT_API
 /* utils.c */
+typedef enum idmap_type_t {
+	ID_TYPE_UID,	/* uidmap entry */
+	ID_TYPE_GID,	/* gidmap entry */
+	ID_TYPE_UIDGID,	/* uidmap and gidmap entry */
+} idmap_type_t;
+
+struct id_map {
+	idmap_type_t map_type;
+	__u32 nsid;
+	__u32 hostid;
+	__u32 range;
+	struct list_head map_head;
+};
```

Comment:
Please, don't use __u32 kernel-ism in userspace, use stdint.h, it means uint32_t in this case.

Reply:
Will do!